portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -112,15 +124,72 @@ function insertSearch() {
 
   const searchInput = document.createElement('input');
   searchInput.className = 'form-control form-inline';
+  searchInput.id = 'userSearch';
   searchInput.type = 'text';
   searchInput.placeholder = 'Search';
 
   const searchI = document.createElement('i');
   searchI.className = 'fas fa-search text-white ml-3 mr-auto';
 
+  const searchResults = document.createElement('ul');
+  searchResults.id = 'searchResults';
+
   searchDiv.appendChild(searchInput);
   searchDiv.appendChild(searchI);
   searchBar.appendChild(searchDiv);
+  searchBar.append(searchResults);
```

Comment:
@kennzoid comment:
https://github.com/kofimeighan/adelardos-capstone-2020/pull/17#discussion_r450619125
Shouldn't this be appendChild too since searchResults is also a DOM element?

Reply:
The results get placed awkwardly if I appendChild them like the other elements.